src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,113 @@
+package duke;
+
+import duke.taskobjects.*;
```

Comment:
Do we need to import everything in `duke.taskobjects`, or is this a wildcard import?